docs/guide/tags/patterns.rst

```diff
@@ -3,79 +3,154 @@
 Tag Patterns
 ============
 
-Tag patterns allow more complex string representation of tags, with a
-notion of if-null defaulting; whilst this sounds complex (and it can be!)
-it's very useful where libraries have wildly varying tags.
+Tag patterns can be used to change the information displayed in various places
+like the playing song area, columns in the song list and the album list in the
+album browser.
 
-Tag patterns can include strings (e.g. tag names) enclosed in angled
-brackets like this
+They can also be used to group songs in the paned and album collection browsers
+in more complex ways.
+
+The file renaming tool uses tag patterns to determine the file names and folder
+structure to be used for the moved files.
+
+Usage
+-----
+
+A tag pattern is a string (piece of text) in which certain symbols receive
+special meaning. Parts of the string that carry no special meaning will be
```

Comment:
```suggestion
special meaning.
Parts of the string that carry no special meaning will be
```

(Generally I'm always trying now to edge towards [sembr](https://sembr.org/) for maintainability.)